src/desktop/config.coffee

```diff
@@ -119,14 +120,14 @@ module.exports =
   STRIPE_PUBLISHABLE_KEY: null
   TEAM_BLOGS: '^\/life-at-artsy$|^\/artsy-education$|^\/buying-with-artsy$'
   TARGET_CAMPAIGN_URL: '/seattle-art-fair-2017'
-  TRACK_PAGELOAD_PATHS: null
+  TRACK_PAGELOAD_PATHS: "home|collect|artist|artwork|show|article|feature|gene|tag|partner|fair|news|new-artwork"
```

Comment:
This risks creating error or warning noise in environments like development and test when Volley (which receives these metrics and sends them to statsd) isn't available.